index.html

```diff
@@ -206,7 +207,15 @@ <h3 class="content-head">Publications/&#x200b;Presentations</h3>
   </script>
 
   <div class="footer l-box is-center">
-      &copy; 2012–2018, Hydra W3C Community Group
+      &copy; 2012–<script language="JavaScript" type="text/javascript">
```

Comment:
```suggestion
      &copy; 2012–<script type="text/javascript">document.write(new Date().getFullYear())</script>
```

Reply:
@alien-mcl Thank you so much for your suggestion, I'll update it.